Makefile

```diff
@@ -97,13 +97,13 @@ build/pyodide.asm.data: root/.built
 
 build/pyodide_dev.js: src/pyodide.js
 	cp $< $@
-	sed -i -e "s#{{DEPLOY}}##g" $@
+	sed -i -e "s#{{DEPLOY}}#./#g" $@
```

Comment:
This fix is necessary to make `pyodide_dev.js` work with nested folders. Without it, `baseURL` in 
https://github.com/iodide-project/pyodide/blob/9f9a6c39656e7adb50970e5b49d4c7de736231bd/src/pyodide.js#L9
becomes `/` and so e.g. `console.html` fails to load when it's put under `/v0.14.3/full/console.html`. With it, the `baseURL` is then `./` and it works fine, at least locally.

Because the cache-control duration is very long for releases, I cannot apply this fix to v0.14.3, and so it means that we can use the new CDN URL only starting from v0.15.0 (and for the deployments from master).